File: src/main/java/com/example/onemoretimejava/basic/PalindromeNumber.java
package com.example.onemoretimejava.basic;

public class PalindromeNumber {
    public static void main(String[] args) {
        isPalindrome(132);
    }

    static void isPalindrome(int num) {
        int temp = num;
        int sum = 0;
        while (num != 0) {
            int n = num % 10;
            sum = sum * 10 + n;
            num = num / 10;

        }
        if (temp == sum) {
            System.out.println("number is palindrome");
        } else {
            System.out.println("number is not palindrome");
        }
    }
}

File: src/main/java/com/example/onemoretimekotlin/basic/PalindromeNumber.kt
package com.example.onemoretimekotlin.basic

fun main(args: Array<String>) {
    println("Hello, world!")

    isPalindrome(131)
}

fun isPalindrome(num: Int) {
    var sum = 0
    var numCopy = num
    while (numCopy != 0) {
        val n = numCopy % 10
        sum = sum * 10 + n
        numCopy /= 10

    }
    if (num == sum) {
        println("is palindrome")
    } else {
        println("is not palindrome")
    }
}